.github/workflows/quality-assurance.yml

```diff
@@ -58,11 +58,19 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        os: [ubuntu-latest, windows-latest]
-        php: [8.3]
-        stability: [prefer-stable]
-
-    name: Run Tests - P${{ matrix.php }} - ${{ matrix.stability }} - ${{ matrix.os }}
+        os:
+          - ubuntu-latest
+          - windows-latest
+        php:
```

Comment:
should we already include [8.4](https://github.com/marketplace/actions/setup-php-action#tada-php-support)? (nightly build)